[2025-01-23T13:31:24.213+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task45_train_models_v2.task234_training_models.task_model3 manual__2025-01-23T13:31:09.673757+00:00 [queued]>
[2025-01-23T13:31:24.283+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task45_train_models_v2.task234_training_models.task_model3 manual__2025-01-23T13:31:09.673757+00:00 [queued]>
[2025-01-23T13:31:24.284+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-23T13:31:24.408+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task234_training_models.task_model3> on 2025-01-23 13:31:09.673757+00:00
[2025-01-23T13:31:24.461+0000] {standard_task_runner.py:60} INFO - Started process 8024 to run task
[2025-01-23T13:31:24.490+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'task45_train_models_v2', 'task234_training_models.task_model3', 'manual__2025-01-23T13:31:09.673757+00:00', '--job-id', '123', '--raw', '--subdir', 'DAGS_FOLDER/task45_train_models_v2.py', '--cfg-path', '/tmp/tmp4gi1x4x6']
[2025-01-23T13:31:24.492+0000] {standard_task_runner.py:88} INFO - Job 123: Subtask task234_training_models.task_model3
[2025-01-23T13:31:24.572+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-23T13:31:24.744+0000] {task_command.py:423} INFO - Running <TaskInstance: task45_train_models_v2.task234_training_models.task_model3 manual__2025-01-23T13:31:09.673757+00:00 [running]> on host a8cdec4ae828
[2025-01-23T13:31:25.115+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task45_train_models_v2' AIRFLOW_CTX_TASK_ID='task234_training_models.task_model3' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T13:31:09.673757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T13:31:09.673757+00:00'
[2025-01-23T13:31:25.604+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-23T13:31:25.622+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=task45_train_models_v2, task_id=task234_training_models.task_model3, execution_date=20250123T133109, start_date=20250123T133124, end_date=20250123T133125
[2025-01-23T13:31:25.696+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-23T13:31:25.741+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
